src/molecules/Modal/ModalFooter.native.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Space from '../../atoms/Space';
+import View from '../../atoms/View';
+import Divider from '../../atoms/Divider';
+
+const ModalFooter = ({ children, type }) => {
+  if (type === 'centered') {
+    return (
+      <Space padding={[2, 3]}>
+        <View>{children}</View>
+      </Space>
+    );
+  }
+  if (type === 'fullscreen') {
```

Comment:
Can you confirm the behavior with Nishant on this as well? All the modals follow a construct which means the footer in this case should also have somehting. It can't be null the API becomes unpredictable since I can write `Modal.Footer` in fullscreen as well and it'll keep returning null